### PR TITLE
GM: Add BSM signal on powertrain bus

### PR DESCRIPTION
**Pull request by garrettpall**
Prerequisite of:
https://github.com/commaai/openpilot/pull/30861

https://github.com/commaai/opendbc/assets/76917194/2a29389a-0717-41c9-9dee-6cd3305baa77

https://github.com/commaai/opendbc/assets/76917194/2b16bda9-4b23-4215-980c-bf8fa24cb94d

### CODE REVIEW

**Comment by ServerDummy** [comment]
Signals match the low-speed signals in a previously accepted PR. It appears though that on the PT bus it sends at a constant rate, unlike lowspeed which only sends when active and one zero bit after it disables.

GMC 1500 route: dc7716b32bf25574|2023-12-27--21-15-59--0

Bolt EUV route: 3140e1c6195c880e|2023-12-27--21-15-14--0


https://github.com/commaai/opendbc/pull/962

File: generator/gm/gm_global_a_powertrain.dbc [hunk 1]
```diff
@@ -105,6 +105,10 @@ BO_ 320 BCMTurnSignals: 3 K9_BCM
  SG_ HighBeamsActive : 7|1@0+ (1,0) [0|1] "" XXX
  SG_ HighBeamsTemporary : 5|1@0+ (1,0) [0|1] "" XXX
 
+BO_ 322 BCMBSM: 7 K9_BCM
```
Comment (sshane) [inline generator/gm/gm_global_a_powertrain.dbc:108]:
```suggestion
BO_ 322 BCMBlindSpotMonitors: 7 K9_BCM
```

little more consistent

File: generator/gm/gm_global_a_powertrain.dbc [hunk 2]
```diff
@@ -105,6 +105,10 @@ BO_ 320 BCMTurnSignals: 3 K9_BCM
...
+ SG_ Left_BSM : 6|1@0+ (1,0) [0|1] "" XXX
+ SG_ Right_BSM : 7|1@0+ (1,0) [0|1] "" XXX
```
Comment (sshane) [inline generator/gm/gm_global_a_powertrain.dbc:110]:
```suggestion
 SG_ LeftBSM : 6|1@0+ (1,0) [0|1] "" XXX
 SG_ RightBSM : 7|1@0+ (1,0) [0|1] "" XXX
```

**Comment by sshane** [comment]
thanks!

**Pull request merged**